templates/base.html

```diff
@@ -137,7 +137,60 @@
         <script src="//cdn.ravenjs.com/3.0.4/raven.min.js"></script>
         <script>Raven.config('{% sentry_public_dsn %}').install()</script>
         {% block script %}{% endblock %}
+        <!-- Piwik -->
+<script type="text/javascript">
+    var _paq = _paq || [];
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+        var u="//analytics.organize.org/";
+        _paq.push(['setTrackerUrl', u+'piwik.php']);
+        _paq.push(['setSiteId', '1']);
+        var gender = '';
+        var birthdate = null;
+        var age = '';
+        {% if cleaned_data %}
+            {% if cleaned_data|key:'gender' == 'M' %}
+                gender = 'Male';
+            {% elif cleaned_data|key:'gender' == 'F' %}
+                gender = 'Female';
+            {% endif %}
 
+            {% if cleaned_data|key:'birthdate' %}
+                birthdate = new Date('{{ cleaned_data|key:'birthdate' }}');
+            {% endif %}
+        {% endif %}
+        if (gender != '') {
+            _paq.push(['setCustomDimension', 2, gender]);
+        }
+        if (birthdate != null) {
+            var ageDifference = Date.now() - birthdate.getTime();
+            var ageDate = new Date(ageDifference); // miliseconds from epoch
+            age = Math.abs(ageDate.getUTCFullYear() - 1970);
+            _paq.push(['setCustomDimension', 1, age]);
```

Comment:
why do we need to calculate this in JS? Why not just fill it out using a template context variable?